### ISSUE TITLE
About train() and no_grad()

### ISSUE THREAD

**Issue by SUN-zx** [city-super/Scaffold-GS#66]
I have observed that in the code, the model's `scene.gaussians.train()` and `gaussians.optimizer.step()` are both performed under the state of `with torch.no_grad():`. Will this not prevent the parameters from being updated? Can parameters be updated solely relying on `loss.backward()`?
